### DISCUSSION TITLE
How can I get student developer pack?

### DISCUSSION THREAD

**Discussion by morbius13** [community/community#62554] [GitHub Education]
I tried everything, used my school email, provided id, set my location and name to real ones and nothing is working. It says they cannot verify that I am a student and that name of the school is not visible in provided documents when it's clearly visible. How to make it work?

**Comment by JoseEmmanuelVG** [comment]
Hello Companion,

I understand the frustration you're experiencing with the GitHub Student Developer Pack verification process. It's essential to meet the specific requirements to qualify for the student benefits. Based on the information I found, here are some important points to consider:

To qualify for the GitHub Student Developer Pack, you must:

- Be currently enrolled in a degree or diploma granting course of study at a high school, secondary school, college, university, homeschool, or similar educational institution.

- Have a verifiable school-issued email address or upload documents that prove your current student status. The name of the school should be clearly visible in the provided documents.

- Be at least 13 years old.

I understand that you have already tried using your school email and provided identification, yet the verification is not successful. In some cases, the verification process might be more straightforward with certain school-issued emails like the one I received from Microsoft for their Azure course (innovaccion.mx). This indicates that different educational institutions might have different verification methods.

One option you can consider is checking if there are any specific courses or programs offered by Microsoft or other companies in your country that provide a school-issued email or documentation for student verification. Taking advantage of such courses not only benefits your learning but also facilitates the verification process for the GitHub Student Developer Pack.

Additionally, it's essential to review the partner company's terms and conditions for availability in your region. Some partner offers may not be available everywhere.

If the issue persists despite trying the above suggestions, I recommend reaching out to GitHub Support for further assistance. They might be able to provide more insights into the verification process and help you resolve the problem.

Remember, the GitHub Student Developer Pack is a valuable resource, and I hope you can overcome the verification hurdles to access its benefits. If there's any other information you need or if I can assist you further, please feel free to let me know.

Best regards,
_JEVG_

**Comment by Awrsha** [comment]
A few things you could try:

Double check that all the information you provided matches exactly what is on your student ID and other documents. Make sure:
Your name, school name, and school email match perfectly
You uploaded high quality, legible photos of your student ID
You provided a recent tuition invoice, registration form, or other enrollment document
Your location is set accurately
Contact GitHub Support directly to explain the situation and ask for help. They may be able to clarify what additional information they need or fix any issues on their end.

Try reapplying from a new GitHub account. Sometimes applications get stuck in a failed state for some reason, and starting fresh can help.

As a last resort, consider asking a parent, teacher, or school administrator to contact GitHub on your behalf. They may have more success getting a manual review of your application due to their official role.

Keep patience and persist politely. GitHub may be facing some internal issues verifying applications, and it can take time for them to investigate and resolve individual cases.

The most important things are to double check that all your information is accurate and verifiable, and then contact GitHub directly for assistance. They will likely need to manually review your application in order to approve it.
